xcube/cli/gen.py

```diff
@@ -135,6 +134,8 @@ def _format_info():
 
     help_text = '\ninput processors to be used with option --proc:\n'
     help_text += _format_input_processors(input_processors)
+    help_text += '\nFor more input processors use existing "xcube-gen-..." plugins ' \
+                 'from the github organisation DCS4COP or write own plugin.\n'
```

Comment:
```suggestion
                 "from xcube's GitHub organisation or write own plugin.\n"
```

Reply:
from the github organisation DCS4COP or write your own plugin.

Reply:
maybe link to teh doczumentation how to write your own plugin?